### ISSUE TITLE
page parameter otherwise broken

### ISSUE THREAD

**Issue by cookie33** [EUDAT-B2HANDLE/B2HANDLE-HRLS#12]
Tested with latest devel branch:
```
$  curl -u "u:p" "https://fqdn:8001/hrls/handles?URL=*&page=1" 
"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'offset 1' at line 1 (SQL error code 1064; SQL State 42000)"

$  curl -u "u:p" "https://fqdn:8001/hrls/handles?URL=*&page=0" 
"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'offset 0' at line 1 (SQL error code 1064; SQL State 42000)"
```
Build on centos7 , run on centos 6

**Comment by cookie33** [comment]
test with war file build on centos 6 system:
```
$  curl -u "u:p" "https://fqdn:8001/hrls/handles?URL=*&page=0" 
"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'offset 0' at line 1 (SQL error code 1064; SQL State 42000)"
```
So it does not matter where the war file is build. It seems broken..

**Comment by cookie33** [comment]
Fixed in #15
